cli/command/container/opts.go

```diff
@@ -364,7 +364,7 @@ func parse(flags *pflag.FlagSet, copts *containerOptions, serverOS string) (*con
 	volumes := copts.volumes.GetMap()
 	// add any bind targets to the list of container volumes
 	for bind := range copts.volumes.GetMap() {
-		parsed, err := loader.ParseVolume(bind)
+		parsed, err := parsevolume.ParseVolume(bind)
```

Comment:
As we're moving this internal, we could take the opportunity to rename, e.g. `volumespec` for the package, and making this `volumespec.Parse()`.